### PR TITLE
solr and disaster recovery

### LINKED ISSUE #5295: Disaster Recovery: Update Solr Limitation
**Note:** Partial or incorrectly filled out issues may be deferred.

Re: https://pantheon.io/docs/disaster-recovery#limitations

Priority: High

## Issue Description

Current "Limitations" section is out of date

## Suggested Resolution
Keep "Limitations" section, update it to something like"

While Solr can be enabled on a site with Disaster Recovery, Pantheon's Solr service is not highly available, nor does it include failover for Solr. The RTO and RPO do not apply to Pantheon's Solr Service.

If your site requires Solr for basic functionality then do not use Pantheon's Solr service. If you require a highly available Solr service, please consider a 3rd-party service, such as....

If your use of Solr is not critical and does not need to include the same assurances as the rest of your site, then you can enable Pantheon's Solr. In that case, Pantheon will need to manually re-provision your Solr instance and you will need to manually re-index Solr after failover.

If you decide to disable Solr, follow our instructions on how to safely remove it from WordPress or Drupal

### PR DESCRIPTION
**Pull request by EdwardAngert**
Closes #5295 

## Effect
PR includes the following changes:
- clarifies conditions surrounding use of Pantheon Solr with Disaster Recovery
- reframes the limitations of Pantheon Solr section of Disaster Recovery to possible solutions

## Remaining Work
- [ ] +1 from @ari-gold 
- [x] @carolyn-shannon do you have any suggestion or a link we can include with L55?
- [ ] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by ari-gold** [comment]
@EdwardAngert  Thanks for putting this together. Let's hold off on this for now. We'll do some testing so we can better set expectations here.  related: CORE-1687

**Comment by carolyn-shannon** [comment]
Important to be clear here we are talking about Pantheon Solr, not Solr as such (which includes externally-hosted Solr, which as an external integration is outside the TOS for DR).

**Comment by carolyn-shannon** [comment]
> Created an issue retroactively that describes what I think we're looking for. Please update to reflect that and let me know if you'd like to discuss

@ari-gold I have updated the Solr text to incorporate your proposed wording, and include secondary index failover option description.

File: source/content/disaster-recovery.md [hunk 1]
```diff
@@ -44,8 +44,27 @@ You can also [connect to your Redis instance](/redis/#use-the-redis-command-line
...
 
-## Limitations
-Solr is not currently supported on DR-enabled sites. See our documentation on how to safely remove Solr for [Drupal 8](/solr-drupal-8/#safely-remove-solr), [Drupal 7](/solr-drupal-7/#safely-remove-solr), and [WordPress](/wordpress-solr/#safely-remove-solr).
+## Solr and Disaster Recovery
+
+### Considerations for Sites Requiring Highly Available Solr Service
+
+While Solr can be enabled on a site with Disaster Recovery, Pantheon's Solr service is not highly available, nor does it include failover for Solr. The RTO and RPO do not apply to Pantheon's Solr Service.
+
+If your site requires Solr for basic functionality, do not use Pantheon's Solr service. If you require a highly available Solr service, please consider an [alternative Solr service](/solr#alternatives-to-pantheons-solr-service).
```
Comment (ari-gold) [inline source/content/disaster-recovery.md]: basic functionality --> . critical site functionality  ?

Comment (alexfornuto) [reply source/content/disaster-recovery.md]: IMHO I'd just mare it "requires Solr".

Comment (alexfornuto) [reply source/content/disaster-recovery.md]: Resolved by https://github.com/pantheon-systems/documentation/pull/5251/commits/5df8c99e6fcc1ecd40dbf192f01efba2e4a930a8

**Pull request merged**